<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Todo lists</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QWidget" name="left_widget" native="true">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="todo_lists">
         <property name="text">
          <string>Todo lists</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListWidget" name="listWidget">
         <property name="selectionMode">
          <enum>QAbstractItemView::ExtendedSelection</enum>
         </property>
         <property name="isWrapping" stdset="0">
          <bool>false</bool>
         </property>
         <property name="resizeMode">
          <enum>QListView::Fixed</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="buttons_widget" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QPushButton" name="addList">
            <property name="text">
             <string>Add list</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="deleteList">
            <property name="text">
             <string>Delete lists</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="right_widget" native="true">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLineEdit" name="header"/>
       </item>
       <item>
        <widget class="QLabel" name="lable_created">
         <property name="text">
          <string>Created:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDateTimeEdit" name="created">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="readOnly">
          <bool>false</bool>
         </property>
         <property name="buttonSymbols">
          <enum>QAbstractSpinBox::NoButtons</enum>
         </property>
         <property name="specialValueText">
          <string/>
         </property>
         <property name="dateTime">
          <datetime>
           <hour>0</hour>
           <minute>0</minute>
           <second>0</second>
           <year>2000</year>
           <month>1</month>
           <day>1</day>
          </datetime>
         </property>
         <property name="displayFormat">
          <string>dd.MM.yyyy HH:mm:ss</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_author">
         <property name="text">
          <string>Author:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="author"/>
       </item>
       <item>
        <widget class="QCheckBox" name="isNumericList">
         <property name="text">
          <string>Numeric list</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTableWidget" name="tableWidget">
         <property name="sortingEnabled">
          <bool>false</bool>
         </property>
         <attribute name="horizontalHeaderVisible">
          <bool>false</bool>
         </attribute>
         <attribute name="verticalHeaderVisible">
          <bool>true</bool>
         </attribute>
         <column>
          <property name="text">
           <string>body</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>status</string>
          </property>
         </column>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="buttons_widget_2" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QPushButton" name="addItem">
            <property name="text">
             <string>Add item</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="deleteItem">
            <property name="text">
             <string>Delete items</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="itemUp">
            <property name="text">
             <string>△</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="itemDown">
            <property name="text">
             <string>▽</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
